specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/databricks.json

```diff
@@ -455,6 +747,39 @@
           "required": [
             "entities"
           ]
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "description": "Private endpoint connections created on the workspace",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "description": "The network access type for accessing workspace. Set value to disabled to access workspace only via private link.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "PublicNetworkAccess"
+          }
+        },
+        "requiredNsgRules": {
+          "type": "string",
+          "description": "Gets or sets a value indicating data plane to control plane communication. Possible values are 'AllRules' and 'NoAzureDatabricksRules'. 'NoAzureServiceRules' value is for internal use only.",
```

Comment:
This value indicates if a Private Endpoint is optional or required for Data Plane (clusters) to Control Plane communication. Supported values are 'AllRules' and 'NoAzureDatabricksRules'. 'NoAzureServiceRules' value is for internal use only.

Reply:
Rephrased description